lib/core/db_models/old/job_data_model.dart

```diff
@@ -26,7 +27,8 @@ class JobDataModel {
     this.issuesArray,
     this.workersArray,
     this.locationsArray,
-    this.totalShiftsCount =0,
+    this.rate,
+    this.totalShiftsCount = 0,
   }) : assert(docID != null);
```

Comment:
assert? are we doing this in the code? what happens? will we trow and catch an exception

Reply:
yes it will through exception if docId is not there moreover this code not added by me